pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1147,7 +1147,7 @@ func (n *NetworkPolicyController) syncAddressGroup(key string) error {
 			// No need to insert Pod IPAddress when it is unset.
 			continue
 		}
-		podSet.Insert(podToMemberPod(pod, true, false))
+		podSet.Insert(podToMemberPod(pod, true, true))
```

Comment:
@jianjuns are you ok with adding pod reference to addressGroup member? guess it will be passed to agents too without extra change in `pkg/controller/networkpolicy/store/addressgroup.go`.

Reply:
this is not needed now that we removed the Pod indexer for the AddressGroup store